index.html

```diff
@@ -38,6 +39,203 @@ <h1 class="text1">
+        <li>2015</li>
+    </ul>
+    </div>
+    
+    <div class="selection">
+        <p>A daily selection of 
+            privately personalized 
+            reads; no accounts or 
+            sign-ups required.</p>
+    </div>
+    <div class="languages">
+        <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+        </ul>
+    </div>
+    <div class="button">
+        <button type="button">See Project</button>
+    </div>
+</section>
+<section class="grid-container">
+<div class="snapshot-image">
+    <img src="images/multy.png">
+</div>
+<div class="tonicmain">
+    <p class="Tonic">Multy-Post<br> Stories</p>
+</div>
+<div class="listitems">
+<ul>
+    <li><span>CANOPY</span></li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>Back End Dev</li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>2015</li>
+</ul>
+</div>
+
+<div class="selection">
+    <P>A daily selection of 
+        privately personalized 
+        reads; no accounts or 
+        sign-ups required.</P>
+</div>
+<div class="languages">
+    <ul>
+        <li><span>html</span></li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+</div>
+    
+<div class="button">
+    <button type="button">See Project</button>
+</div>
+</section>
+<section class="grid-container">
+<div class="snapshot-image">
+    <img src="images/tonic.png" alt="tonic-Page">
+</div>
+<div class="tonicmain">
+    <p>Tonic</p>
+</div>
+<div class="listitems">
+<ul>
+    <li><span>Canopy</span></li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>Back End Dev</li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>2015</li>
+</ul>
+</div>
+
+<div class="selection">
+    <p>A daily selection of 
+        privately personalized 
+        reads; no accounts or 
+        sign-ups required.</p>
+</div>
+<div class="languages">
+    <ul>
+        <li><span>html</span></li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+</div>
+    
+<div class="button">
+    <button type="button">See Project</button>
+</div>
+</section>
+<section class="grid-container">
+<div class="snapshot-image">
+    <img src="images/multypost.png" alt="">
+</div>
+<div class="tonicmain">
+    <p>Tonic</p>
+</div>
+<div class="listitems">
+<ul>
+    <li><span>CANOPY</span></li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>Back End Dev</li>
+    <li><img src="images/Counter.png" alt=""></li>
+    <li>2015</li>
+</ul>
+</div>
+
+<div class="selection">
+    <p>A daily selection of 
+        privately personalized 
+        reads; no accounts or 
+        sign-ups required.</p>
+</div>
+<div class="languages">
+    <ul>
+        <li><span>html</span></li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+</div>
+    
+<div class="button">
+    <button type="button">See Project</button>
+</div>
+</section>
+<section class="grid-container">
+<div class="tonicmain">
+    <p>About Myself</p>
```

Comment:
- [ ]  Kindly ensure that the about me section title (e.g., "About myself") uses a header tag `hx`( i.e I will recommend you use the `<h2>` tag) just as the previous reviewer suggested👍🏽